lib/tasks/maintenance.rake

```diff
@@ -43,4 +43,38 @@ namespace :maintenance do
       from_stage.destroy! if delete_empty && from_stage.deploy_groups.empty?
     end
   end
+
+  # FROM and TO handles multiple deploy groups to not mess with projects that have each deploy group in a single stage.
+  desc "delete deploy groups across stages for all projects. OWNER= DELETE=dg1,dg2 [DELETE_EMPTY=true]"
+  task delete_deploy_group: :environment do
+    projects = Project.where(owner: ENV.fetch("OWNER"))
+    delete = ENV.fetch("DELETE").split(",")
+    delete_group = DeployGroup.find_by_permalink!(delete)
```

Comment:
if we pass multiple with comma, how will it fork? IS delete_group would be an array then?

Reply:
it wouldn't have but we'll do one group at a time for safety anyways